### DISCUSSION TITLE
Change creation date from admin panel

### DISCUSSION THREAD

**Discussion by achristop** [strapi/strapi#8639] [Legacy Topics]
Hi,
I would like to know if it is possible to have a date field in the admin panel and change the creation date of a content type for example an article ?

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).

**Reply by achristop** [reply]
Thank you, i ll ask in the forum.
